helios-docs/docs/QuickStart.md

```diff
@@ -0,0 +1,64 @@
+# QuickStart
+
+Once added helios to you project, let's start to add the needed imports:
+
+```kotlin:ank:silent
+import arrow.core.*
+import helios.core.*
+import helios.meta.*
+import helios.optics.*
+import helios.typeclasses.*
+```
+
+Now, we can start to create our DSL.
+
+## DSL
+
+```kotlin
+@json
+data class Person(val name: String, val age: Int) {
+  companion object
+}
+```
+
+The `@json` annotation will provide the decoder and encoder for that data class, so we are able to
+
+## Decode
+
+We can decode from a `String`, a `File`, etc
+
+```kotlin:ank
+val jsonStr = 
+"""{
+     "name": "Simon",
+     "age": 30
+   }"""
+    
+val jsonFromString : Json = 
+  Json.parseFromString(jsonStr).getOrHandle {
+    println("Failed creating the Json ${it.localizedMessage}, creating an empty one")
+    JsString("")
+  }
+
+val personOrError: Either<DecodingError, Person> = Person.decoder().decode(jsonFromString)
+
+personOrError.fold({
+  "Something went wrong during decoding: $it"
+}, {
+  "Successfully decode the json: $it"
+})
+```
+
+## Encode
+
+We can also encode from a data class instance to a `Json`
+
+```kotlin:ank
+val person = Person("Raul", 34)
+
+val jsonFromPerson = with(Person.encoder()) {
+  person.encode()
+}
+
+println(jsonFromPerson.toJsonString())
```

Comment:
If you use this style of ank snippets println returns unit and the result of every expression is going to be unit